paas2/paas/user_center/wx_utils.py

```diff
@@ -14,22 +14,22 @@
 
 from components import usermgr
 from common.log import logger
-from esb.bkcore.models import ESBChannel
 from user_center.constants import WxTypeEnum
 from user_center.models import WxBkUserTmpRecord
+from blueking.component.shortcuts import get_client_by_user
 
 
 def get_wx_config():
     """
     获取微信相关配置
     """
-    wx_comp_path = "/cmsi/send_weixin/"
-    # 检查是否配置了微信通知组件
-    if not ESBChannel.objects.filter(path=wx_comp_path).exists():
+    client = get_client_by_user("")
```

Comment:
get_client_by_user  username param not empty string,  like "admin"

Reply:
cahnge to "admin"